engine/src/test/java/io/zeebe/engine/processor/workflow/deployment/model/validation/ZeebeRuntimeValidationTest.java

```diff
@@ -145,6 +146,14 @@
             .done(),
         Arrays.asList(expect(ZeebeLoopCharacteristics.class, INVALID_PATH_QUERY_MESSAGE))
       },
+      {
+        // output element expression is not supported
+        Bpmn.createExecutableProcess("process")
+            .startEvent()
+            .serviceTask("task", t -> t.zeebeJobType(INVALID_EXPRESSION))
+            .done(),
+        Arrays.asList(expect(ZeebeTaskDefinition.class, INVALID_EXPRESSION_MESSAGE))
```

Comment:
Using the job type expression builder instead of the static one.

```suggestion
        // invalid job type expression
        Bpmn.createExecutableProcess("process")
            .startEvent()
            .serviceTask("task", t -> t.zeebeJobTypeExpression(INVALID_EXPRESSION))
            .done(),
        Arrays.asList(expect(ZeebeTaskDefinition.class, INVALID_EXPRESSION_MESSAGE))
```